stackdriver/adapter_test.go

```diff
@@ -17,34 +17,32 @@ package stackdriver
 import (
 	"context"
 	"fmt"
+
 	"os"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/google/ts-bridge/mocks"
-
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
-	"google.golang.org/appengine/aetest"
+	"github.com/google/ts-bridge/datastore"
+	"github.com/google/ts-bridge/mocks"
 	metricpb "google.golang.org/genproto/googleapis/api/metric"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
-var testCtx context.Context
+var ctx context.Context
```

Comment:
This is probably bad. You should probably never have a global variable containing a context. If you do, then I think that context is being misused

Reply:
Yeah agreed. Removed in favour of separate contexts.